### PR TITLE
Move searchbar's position to sidebar

### PR DESCRIPTION
**Pull request by Demolemon11**
#### Before:
Screenshot from 2024-10-13 10-14-21

#### After:
Screenshot from 2024-10-13 10-37-04

### CODE REVIEW

**Review by Demolemon11 — COMMENTED** [review]
Please view : )

**Comment by joulei** [comment]
@Demolemon11 this actually a design decision, we're currently discussing with a designer what this will look like.
Furthermore, there's already a PR adding a local search to filter out the rooms on left panel: https://github.com/project-robius/robrix/pull/159.
Also, there's a limitation with Makepad's implementation of the window title in some platforms where if you attempt to move the tabs around (without having some other space above like the search bar), the whole window will move instead of the tab.

TLDR: I don't think we should do this change right now. Let's us solve some of the issues around it first.
(we also have some things to discuss like what will the rooms sidebar search look like with added People and Channels sections).

**Comment by Demolemon11** [comment]
Got it, thinks so much.

**Comment by kevinaboos** [comment]
Agreed, I don't think this direction is desirable at this moment. Apologies, but thanks for the work!
